Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.3
```

Comment:
I updated `swift-tools-version` to 5.3 because I would to use `resources` in `.target(name: , path: , resources)`.
This is required to publish the resource file externally.